bones/numericBone.py

```diff
@@ -48,15 +48,14 @@ def getEmptyValue(self):
 			return 0
 
 	def isEmpty(self, rawValue: Any):
-		if isinstance(rawValue, str) and not rawValue:
-			return True
-		try:
-			if self.precision:
-				rawValue = float(rawValue.replace(",", ".", 1))
-			else:
-				rawValue = int(rawValue.replace(",", ".", 1))
-		except:
-			return True
+		if isinstance(rawValue, str):
+			try:
+				if self.precision:
+					rawValue = float(rawValue.replace(",", ".", 1))
+				else:
+					rawValue = int(rawValue)
```

Comment:
I think we should also accept here strings like `"123,89"` and round them down to a int. This would be the analogous behavior if the precision is a positiv number and you supplied more fractional digits than allowed (see [l. 69](https://github.com/viur-framework/viur-core/blob/e7ec7bc67776250f70d22bc5e5e67204aa294b95/bones/numericBone.py#L69) for this).

```suggestion
					rawValue = int(float(rawValue.replace(",", ".", 1)))
```